tools/packaging/kata-deploy/kata-deploy/components/stable/env_stable_conf.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata: 
-  name: kata-deploy
-  namespace: kube-system
+  name: not-important
+  namespace: not-important
```

Comment:
Why? :-)

Reply:
If you're using overlays, Kustomize may ignore changes in name or namespace specified in the overlay if they are not "properly" configured. The base configuration often takes precedence unless the overlay explicitly modifies these fields.

Reply:
By modifying adding a prefix or suffix explicitly but since this is a selector nothing gets changed.